packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -863,6 +863,93 @@ describe('serverless cluster', () => {
 
 
   });
+
+  test('can create a Serverless cluster without vpc', () => {
```

Comment:
Let's spell it "VPC" in all of these new tests, not "vpc".

This comment applies everywhere in the PR, not only to this line.